### ISSUE TITLE
no-unused-styles: False positive when using dot access on style.

### ISSUE THREAD

**Issue by joshuapinter** [Intellicode/eslint-plugin-react-native#293]
For example: 

```js
width={ styles.thumbnail.width }
```

This will raise an error with the following style: 

```js
thumbnail: {
  width: "30%"
},
```

```
eslint: error
react-native/no-unused-styles - Unused style detected: styles.thumbnail
```

Using dot notation is ignored when determining which defined styles are used.
